e2e/test_operator/operator_test.go

```diff
@@ -124,6 +126,274 @@ var _ = Describe("Operator", Label("e2e"), func() {
 		}
 	})
 
+	FContext("testing taint feature", func() {
+		taintKeyMaintenance := "maintenance"
+		taintKeyMaintenanceDuration := int64(2)
+		taintKeyStar := "*"
+		taintKeyStarDuration := int64(5)
+		var initialPods *corev1.PodList
+		var taintedNode *corev1.Node
+		var taintedNodes []*corev1.Node
+		numNodesTainted := 3 // TODO: Change to higher number 5
+		ensurePodIsDeletedTimeoutMinutes := 20
+
+		BeforeEach(func() {
+			initialPods = fdbCluster.GetStatelessPods()
+
+			// Setup cluster's taint config and wait for long enough
+			curClusterSpec := fdbCluster.GetCluster().Spec.DeepCopy()
+			curClusterSpec.AutomationOptions.Replacements.TaintReplacementOptions = []fdbv1beta2.TaintReplacementOption{
```

Comment:
Any reason we are not using `fdbCluster.SetClusterTaintConfig` here?

Reply:
`SetClusterTaintConfig`  is just a wrapper of `UpdateClusterSpecWithSpec`. 
I added `SetClusterTaintConfig ` first and then realized I can do the same with `UpdateClusterSpecWithSpec`.

I'm leaning to remove `SetClusterTaintConfig`.

What do you think?